lib/rules/function-linear-gradient-no-nonstandard-direction/index.js

```diff
@@ -63,7 +63,7 @@ const rule = (primary) => {
 
 				functionArgumentsSearch(
 					valueParser.stringify(valueNode).toLowerCase(),
-					'linear-gradient',
+					/^(-webkit-|-moz-|-o-)?linear-gradient$/i,
```

Comment:
[note] `postcss-value-parser` doesn't seem to support the fuzzy match. Instead, using regex.